astropy/extern/README.rst

```diff
@@ -96,7 +95,8 @@ configuration module::
 Third-party packagers can override the defaults for these configuration items
 (by modifying the configuration objects in ``astropy/table/jsviewer.py``, or
 provide astropy config files that include the overrides appropriate for the
-packaged version.
+packaged version.  They would *also* need to set the default
+``use_local_files`` option to ``False`` for these settings to be read.
```

Comment:
I'm confused--this note doesn't make sense anymore because there is no `use_local_files` option?